app/assets/src/components/views/visualizations/VisualizationsView.jsx

```diff
@@ -72,6 +98,11 @@ class VisualizationsView extends React.Component {
     );
   };
 
+  handleRowClick = ({ event, index, rowData }) => {
```

Comment:
Its good not the variable `event` as it a global variable (aka `window.event`).
To avoid possible future bugs, rename it to something else like `currentEvent`.

Reply:
Good point. FYI, it is also named `event` in `ProjectsView` and `SamplesView`.

Reply:
Thanks for pointing that out. Should not.